src/components/pages/Categories/Edit.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <app-category-edit
+    :access="access"
+    :category="category"
+    :category-id="categoryId"
+    :current-category-name="currentCategoryName"
+    :disabled="disabled"
+    :done-message="doneMessage"
+    :error-message="errorMessage"
+    @update-value="updateValue"
+    @edit-category="editCategory"
+  />
+</template>
+
+<script>
+import { CategoryEdit } from '@Components/molecules';
+
+export default {
+  components: {
+    appCategoryEdit: CategoryEdit,
+  },
+  computed: {
+    access() {
+      return this.$store.getters['auth/access'];
+    },
+    category() {
+      return this.$store.state.categories.category.name;
+    },
+    categoryId() {
+      let { id } = this.$route.params;
+      id = parseInt(id, 10);
+      return id;
```

Comment:
[imo]

なるべくシンプルにしましょう

```suggestion
      return parseInt(this.$route.params.id, 10);
```